lamindb_setup/core/_setup_bionty_sources.py

```diff
@@ -67,50 +54,50 @@ def _get_currently_used(key: str):
     Source.objects.bulk_create(all_records, ignore_conflicts=True)
 
 
-def load_bionty_sources(isettings: InstanceSettings | None = None):
-    """Write currently_used bionty sources to LAMINDB_VERSIONS_PATH in bionty."""
-    if isettings is None:
-        if setup_settings._instance_settings is not None:
-            isettings = setup_settings.instance
-        else:
-            logger.warning(
-                f"Ignoring bionty setup because running in LAMINDB_MULTI_INSTANCE mode = {os.environ.get('LAMINDB_MULTI_INSTANCE')}"
-            )
-            # not setting up bionty sources
-            return None
-    if isettings is not None:
-        if "bionty" not in isettings.modules:
-            # no need to setup anything
-            return None
-
-    import bionty.base as bionty_base
-    from bionty.base.dev._handle_sources import parse_currently_used_sources
-    from bionty.base.dev._io import write_yaml
-    from bionty.models import Source
-
-    try:
-        # need try except because of integer primary key migration
-        active_records = (
-            Source.objects.filter(currently_used=True).order_by("id").all().values()
-        )
-        for kwargs in active_records:
-            for db_field, base_col in RENAME.items():
-                kwargs[base_col] = kwargs.pop(db_field)
-            # TODO: non-bionty modules?
-            kwargs["entity"] = kwargs["entity"].replace("bionty.", "")
-        write_yaml(
-            parse_currently_used_sources(active_records),
-            bionty_base.settings.lamindb_sources,
-        )
-    except (OperationalError, ProgrammingError):
-        pass
-
-
-def delete_bionty_sources_yaml():
-    """Delete LAMINDB_SOURCES_PATH in bionty."""
-    try:
-        import bionty.base as bionty_base
-
-        bionty_base.settings.lamindb_sources.unlink(missing_ok=True)
-    except ModuleNotFoundError:
-        pass
+# def load_bionty_sources(isettings: InstanceSettings | None = None):
```

Comment:
Why not just remove this instead of commenting out?

Reply:
I will! Currently just testing out :D